dist/chota.css

```diff
@@ -629,6 +629,21 @@ button:disabled:hover {
 ::placeholder {
   color: #bdbfc4;
 }
+.group {
+	display: -webkit-box;
+	display: -ms-flexbox;
+	display: flex; /* hide space between element */
+}
+.group>* {
+	margin: 0 !important;
+	border-radius:0 !important;
+}
+.group>*:first-child{
+	border-radius:4px 0 0 4px !important;
+}
+.group>*:last-child{
+	border-radius:0 4px 4px 0 !important;
+}
```

Comment:
we need a `margin-left: -1px` for the elements to eliminate the double border between them

Reply:
good catch

Reply:
the way to handle that it with a `z-index` on the element that has the strongest border

Reply:
Nice, I've added `z-index:1` to `.button.primary` and it's working perfectly.